installers-conda/resources/post-install.sh

```diff
@@ -13,29 +13,34 @@ echo ""
 name_lower=$(echo ${INSTALLER_NAME} | tr 'A-Z' 'a-z')
 spy_exe=${PREFIX}/envs/spyder-runtime/bin/spyder
 u_spy_exe=${PREFIX}/uninstall-spyder.sh
+all_user=$([[ -e ${PREFIX}/.nonadmin ]] && echo false || echo true)
 
-sed_opts=("-i")
 alias_text="alias uninstall-spyder=${u_spy_exe}"
-if [[ $OSTYPE = "darwin"* ]]; then
-    shortcut_path="/Applications/Spyder.app"
-    [[ ${PREFIX} = "$HOME"* ]] && shortcut_path="${HOME}${shortcut_path}"
+if [[ "$OSTYPE" = "darwin"* ]]; then
+    shortcut_path="/Applications/${INSTALLER_NAME}.app"
+    if [[ "$all_user" = "false" ]]; then
+        shortcut_path="${HOME}${shortcut_path}"
+        unset alias_text  # Do not create uninstall alias
+    fi
     sed_opts+=("", "-e")
 else
-    shortcut_path="$HOME/.local/share/applications/${name_lower}_${name_lower}.desktop"
-    alias_text="alias spyder=${spy_exe}\n${alias_text}"
+    shortcut_path="/share/applications/${name_lower}_${name_lower}.desktop"
+    if [[ "$all_user" = "true" ]]; then
+        shortcut_path="/usr${shortcut_path}"
```

Comment:
```suggestion
        shortcut_path="/usr/local${shortcut_path}"
```

Reply:
What is the name of the desktop file created by the Linux distro package manager?
Couple of points:
* `menuinst` names the desktop file `spyder_spyder.desktop`. We have control over the app name, e.g. "spyder", but `menuinst` will always name the file `<app-name>_<app_name>.desktop`.
* `menuinst` will always place the shortcut in `/usr/share/applications` or `$HOME/.local/share/applications` and I don't think we have any control over that.

So, I don't think your suggestion is possible to do, but if desktop files don't have a name conflict, then perhaps there isn't a problem.

@jaimergp, am I mistaken about any of the above?

Reply:
That's correct. It's the slugified names ([see here](https://github.com/conda/menuinst/blob/23c580b5d798db157ced01cc11278c1436572d3b/menuinst/platforms/linux.py#L184-L189)) and the XDG standard paths ([see here](https://github.com/conda/menuinst/blob/23c580b5d798db157ced01cc11278c1436572d3b/menuinst/platforms/linux.py#L28-L55)).

Reply:
> What is the name of the desktop file created by the Linux distro package manager?

I check in Ubuntu and it's `spyder.desktop`:

https://packages.ubuntu.com/kinetic/all/spyder/filelist

So, we should be fine.

> menuinst will always place the shortcut in /usr/share/applications

Ok, I didn't know that. Then please disregard my suggestion.